### DISCUSSION TITLE
SMS authentication

### DISCUSSION THREAD

**Discussion by drilnet** [community/community#64845] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello (I have bad English)!

Settings => Password and authentication => SMS authentication.
I can not select the country Ukraine?
How to fix it? Who can be contacted?

Thank you.
:(

**Comment by w8ste** [comment]
@drilnet According to this [list](https://docs.github.com/en/enterprise-cloud@latest/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported), which lists all of the countries SMS authentication is supported for, Ukraine is sadly not one of them. I don't think there is a lot you can to about it, except hope they will change it soon. Best of luck :)

**Comment by drilnet** [comment]
Github requires two-level authentication!
Two authentication options (and they don't work)!
:(

I need help.

My site: https://drilnet.github.io/

Thank.
